openpype/hosts/unreal/hooks/pre_workfile_preparation.py

```diff
@@ -111,6 +112,7 @@ def exec_ue_project_gen(self,
         ue_project_worker = UEProjectGenerationWorker()
         ue_project_worker.setup(
             engine_version,
+            self.project_name,
```

Comment:
You don't need to store it to attribute of the hook.
```suggestion
            self.data["project_name"],
```